corehq/apps/app_manager/suite_xml/sections/details.py

```diff
@@ -69,13 +67,13 @@ def get_section_elements(self):
         return r
 
     def build_detail(self, module, detail_type, detail, detail_column_infos,
-                     tabs, id, title, start, end):
+                     tabs=[], id=None, title=None, nodeset=None, start=0, end=None):
```

Comment:
It's dangerous to use mutable objects as default arguments. This method doesn't mutate `tabs`, but someone might do so in the future thinking it's safe. Looks like you could use `None` here instead without needing to change anything else in the method.